Pylint warnings on `runtime`

`tinygrad/runtime` is missing an `__init__.py` file which means `pylint` won't lint it.

```
************* Module tinygrad.runtime.ops_python
tinygrad/runtime/ops_python.py:19:2: R1705: Unnecessary "else" after "return", remove the "else" and de-indent the code inside it (no-else-return)
tinygrad/runtime/ops_python.py:135:23: W0640: Cell variable inp defined in loop (cell-var-from-loop)
tinygrad/runtime/ops_python.py:136:23: W0640: Cell variable inp defined in loop (cell-var-from-loop)
tinygrad/runtime/ops_python.py:137:23: W0640: Cell variable inp defined in loop (cell-var-from-loop)
tinygrad/runtime/ops_python.py:138:31: W0640: Cell variable inp defined in loop (cell-var-from-loop)
tinygrad/runtime/ops_python.py:139:31: W0640: Cell variable inp defined in loop (cell-var-from-loop)
tinygrad/runtime/ops_python.py:140:31: W0640: Cell variable inp defined in loop (cell-var-from-loop)
tinygrad/runtime/ops_python.py:142:19: W0640: Cell variable inp defined in loop (cell-var-from-loop)
tinygrad/runtime/ops_python.py:147:60: W0640: Cell variable inp defined in loop (cell-var-from-loop)
tinygrad/runtime/ops_python.py:147:92: W0640: Cell variable inp defined in loop (cell-var-from-loop)
tinygrad/runtime/ops_python.py:163:46: W1114: Positional arguments appear to be out of order (arguments-out-of-order)
tinygrad/runtime/ops_python.py:163:46: W0640: Cell variable a_elem defined in loop (cell-var-from-loop)
************* Module tinygrad.runtime.ops_cuda
tinygrad/runtime/ops_cuda.py:121:4: R1705: Unnecessary "else" after "return", remove the "else" and de-indent the code inside it (no-else-return)
tinygrad/runtime/ops_cuda.py:124:4: R1705: Unnecessary "else" after "return", remove the "else" and de-indent the code inside it (no-else-return)
tinygrad/runtime/ops_cuda.py:10:20: W0611: Unused import extra.nv_gpu_driver.nv_ioctl (unused-import)
************* Module tinygrad.runtime.ops_nv
tinygrad/runtime/ops_nv.py:333:4: R1705: Unnecessary "else" after "return", remove the "else" and de-indent the code inside it (no-else-return)
tinygrad/runtime/ops_nv.py:336:2: W0237: Parameter 'opaque' has been renamed to 'gpumem' in overriding 'NVAllocator._free' method (arguments-renamed)
************* Module tinygrad.runtime.ops_hsa
tinygrad/runtime/ops_hsa.py:50:30: W0707: Consider explicitly re-raising using 'raise CompileError(e) from e' (raise-missing-from)
tinygrad/runtime/ops_hsa.py:108:4: R1705: Unnecessary "else" after "return", remove the "else" and de-indent the code inside it (no-else-return)
tinygrad/runtime/ops_hsa.py:261:69: E0203: Access to member 'kernarg_start_addr' before its definition line 262 (access-member-before-definition)
tinygrad/runtime/ops_hsa.py:10:20: W0611: Unused import extra.hip_gpu_driver.hip_ioctl (unused-import)
************* Module tinygrad.runtime.ops_metal
tinygrad/runtime/ops_metal.py:19:4: R1705: Unnecessary "else" after "return", remove the "else" and de-indent the code inside it (no-else-return)
tinygrad/runtime/ops_metal.py:27:34: W0707: Consider explicitly re-raising using 'raise CompileError(e) from e' (raise-missing-from)
************* Module tinygrad.runtime.ops_npy
tinygrad/runtime/ops_npy.py:5:0: W0223: Method '_alloc' is abstract in class 'Allocator' but is not overridden in child class 'NpyAllocator' (abstract-method)
tinygrad/runtime/ops_npy.py:5:0: W0223: Method 'copyin' is abstract in class 'Allocator' but is not overridden in child class 'NpyAllocator' (abstract-method)
************* Module tinygrad.runtime.ops_amd
tinygrad/runtime/ops_amd.py:59:30: W0707: Consider explicitly re-raising using 'raise CompileError(e) from e' (raise-missing-from)
tinygrad/runtime/ops_amd.py:349:6: R1705: Unnecessary "else" after "return", remove the "else" and de-indent the code inside it (no-else-return)
tinygrad/runtime/ops_amd.py:355:2: W0237: Parameter 'opaque' has been renamed to 'gpumem' in overriding 'AMDAllocator._free' method (arguments-renamed)
************* Module tinygrad.runtime.ops_disk
tinygrad/runtime/ops_disk.py:21:2: W0237: Parameter 'opaque' has been renamed to 'buf' in overriding 'DiskAllocator._free' method (arguments-renamed)
************* Module tinygrad.runtime.ops_gpu
tinygrad/runtime/ops_gpu.py:65:4: R1705: Unnecessary "else" after "return", remove the "else" and de-indent the code inside it (no-else-return)
tinygrad/runtime/ops_gpu.py:70:2: W0237: Parameter 'opaque' has been renamed to 'buf' in overriding 'CLAllocator._free' method (arguments-renamed)

------------------------------------------------------------------
Your code has been rated at 9.94/10 (previous run: 9.94/10, +0.00)
```

--- a/tinygrad/runtime/ops_python.py
+++ b/tinygrad/runtime/ops_python.py
@@ -1,4 +1,4 @@
-# pylint: skip-file
+# pylint: disable=cell-var-from-loop
 # a python uops emulator
 # works to test the tensor cores, and all the uops in general
 # this is the (living) definition of uops
@@ -164,7 +164,7 @@ class PythonProgram:
               assert x[i][goff+j] == x[i][goff+j+16], "warp elements not duplicated properly across lanes"
               return x[i][goff+j]
             # B (16 elements on 32 threads): row major, lane 16-32 == lane 0-15
-            def b_elem(x, i, j, goff): return a_elem(x, j, i, goff)
+            def b_elem(x, i, j, goff): return a_elem(x, j, i, goff)  # pylint: disable=arguments-out-of-order
             def c_map(lane, elem): return (lane%16, lane//16+elem*2) # (i, j), C, D (8 elements on 32 threads): row major
             ul[i] = wmma_helper(32, 16, 16, 16, 8, a_elem, b_elem, c_map)
           elif arg[5] == "CUDA":

